defaults.env

```diff
@@ -0,0 +1,4 @@
+POSTGRES_DB=link_db
+POSTGRES_USER=link_user
+POSTGRES_PASSWORD=super-secure-password
+DATABASE_URL=postgresql://link_user:super-secure-password@postgres/link_db
```

Comment:
maybe we can create a sub directory called "postgresql" and rename this to `postgresql/defaults.env.example` ?